spec/reek/rake/task_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../spec_helper'
+require_lib 'reek/rake/task'
+
+RSpec.describe Reek::Rake::Task do
+  describe '#initialize' do
+    it 'works' do
+      Reek::Rake::Task.new
+    end
+  end
+
+  describe '#source_files' do
+    it 'is set to "lib/**/*.rb" by default' do
+      task = Reek::Rake::Task.new
+      expect(task.source_files).to eq FileList['lib/**/*.rb']
+
+    end
+  end
+
+  describe '#source_files=' do
+    it 'sets source_files to a FileList when passed a string' do
+      task = Reek::Rake::Task.new
+      task.source_files = '*.rb'
+      expect(task.source_files).to eq FileList['*.rb']
+    end
+  end
+
+  describe '#command' do
+    let(:task) { Reek::Rake::Task.new }
+
+    it 'does not include a config file by default' do
+      expect(task.command).not_to include '-c'
```

Comment:
Hmm, so you made this public just for the test, now i see.
I'd then rather use `send` ....